src/dbnode/topology/dynamic.go

```diff
@@ -73,6 +74,25 @@ func (i *dynamicInitializer) Init() (Topology, error) {
 	return i.topo, nil
 }
 
+func (i *dynamicInitializer) TopologySet() (bool, error) {
```

Comment:
Is this going to make an RPC to KV each time?

Reply:
Correct, we can live with this for now (also existing k8s operator was calling the operator endpoint to simulate this behavior already).